### PR TITLE
{lang} [foss-2015a,intel-2015a] Python (REVIEW) 

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
I'm going to need this for xmipp

File: easybuild/easyconfigs/p/Python/Python-2.7.9-foss-2015a-bare.eb [hunk 1]
```diff
@@ -17,8 +17,7 @@ dependencies = [
     ('zlib', '1.2.8'),
     ('libreadline', '6.3'),
     ('ncurses', '5.9'),
-#   ('OpenSSL', '1.0.1k'),  # OS dependency should be preferred if the os version is more recent then this version, it's
-#   nice to have an up to date openssl for security reasons
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.9-foss-2015a-bare.eb]: keep this please, people should be able to easily switch to an EB-provided OpenSSL

**Comment by boegel** [comment]
There currently are *7* easyconfigs for Python 2.7.9, please tackle all 7 (and add the missing `SQLite` easyconfigs).

**Comment by JensTimmerman** [comment]
this isn't going to work

**Comment by boegel** [comment]
What's the problem exactly?

**Comment by boegel** [comment]
Ah, not your branch, so no control. OK.

**Comment by JensTimmerman** [comment]
folow up in https://github.com/hpcugent/easybuild-easyconfigs/pull/1468

**Pull request closed without merging**